swagger.json

```diff
@@ -219,22 +219,7 @@
       ]
     },
     "udf_description": {
-      "description": "Defines and describes a UDF using the same schema as the description of processes offered by the back-end.",
```

Comment:
When using $ref all other arguments are ignored. If we want the custom description and example we need to copy over the other content from the referenced definition.